index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; URL= https://austinbwanika.github.io/ugandaemr-technicalguide/SUMMARY.html" >
```

Comment:
@AustinBwanika Can you remove the https://austinbwanika.github.io/ugandaemr-technicalguide/ from this url

Reply:
@ssmusoke I have changed the URL but it goes to the repository instead of the site.

Reply:
This is because you changed the url to point to the repository, the correct url should be "URL=SUMMARY.html"

Reply:
@ssmusoke Now it is working but the README file isn't be rendered into HTML

Reply:
@AustinBwanika check the deployment logs to find out why that is happening

Reply:
@ssmusoke Okay I will check them to find out the error